client/app/components/packages/rwcds-form/proposed-project-actions-editor.js

```diff
@@ -18,7 +18,7 @@ export default class ProposedActionsComponent extends Component {
 
   get zrAppendixF() {
     const zrSectionNumber = this.args.zrForm.saveableChanges.dcpZrsectionnumber;
-    return this.zrTypeLabel === 'ZR' && zrSectionNumber === 'Appendix F';
+    return this.zrTypeLabel === 'ZR' && zrSectionNumber === 'AppendixF';
```

Comment:
Let's use a lookup for this kind of thing — don't necessarily have to use optionset helper, just a dictionary will do

Reply:
I'm gonna vouch for  `{{optionset}}` helper all the things :p to avoid errors like this.
Optionset helper will spit out a console assertion error if the passed in identifier/code/label doesn't exist.